### ISSUE TITLE
Update runtime to 5.12

### ISSUE THREAD

**Issue by LeandroStanger** [flathub/org.openscad.OpenSCAD#3]

**Comment by goncalossilva** [comment]
I've played with this, but CGAL fails to build due to failing to find MPFR. This works fine in 5.11.

```
== Detect external libraries ==
-- External libraries supported: GMP;GMPXX;MPFR;ZLIB;OpenGL;LEDA;MPFI;RS;RS3;OpenNL;Eigen3;BLAS;LAPACK;QGLViewer;ESBTL;Coin3D;NTL;IPE
-- Preconfiguring library: GMP ...
-- Found GMP: /usr/lib/x86_64-linux-gnu/libgmp.so  
-- GMP has been preconfigured:
--   UseGMP-file:      
--   GMP include:      /usr/include
--   GMP libraries:    /usr/lib/x86_64-linux-gnu/libgmp.so
--   GMP definitions:  
-- USING GMP_VERSION = '6.1.2'
-- Preconfiguring library: MPFR ...
-- Could NOT find MPFR (missing: MPFR_LIBRARIES MPFR_INCLUDE_DIR) 
CMake Error at cmake/modules/CGAL_SetupDependencies.cmake:66 (message):
  CGAL requires MPFR to be found
Call Stack (most recent call first):
  CMakeLists.txt:671 (include)
```

**Comment by t-paul** [comment]
Closed via #6 (updating to 5.13).
